feature-libs/quote/root/guards/quote-cart.service.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Observable, ReplaySubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class QuoteCartService {
```

Comment:
`minor`: I propose creating a ticket about removing this service once the backend is ready and adding comments with the ticket number to all related places. It may be useful in terms of tracking and refactoring.